data/src/main/java/ca/bc/gov/data/datasource/local/dao/MedicationRecordDao.kt

```diff
@@ -20,6 +20,6 @@ interface MedicationRecordDao : BaseDao<MedicationRecordEntity> {
     @Query("SELECT * FROM medication_record WHERE id = :medicationRecordId")
     suspend fun getMedicationWithSummaryAndPharmacy(medicationRecordId: Long): MedicationWithSummaryAndPharmacy?
 
-    @Query("DELETE FROM medication_record WHERE patient_id = :patientId AND data_source = 'BCSC'")
+    @Query("DELETE FROM medication_record WHERE patient_id = :patientId")
     suspend fun deleteAuthenticatedMedicationRecords(patientId: Long): Int
```

Comment:
rename method to delete()

Reply:
renamed to deletePatientTestRecords() as delete already exists.